spec/factories/user.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :user do
+    email { "jean@dfsseta.gov.uk" }
+    encrypted_password { "password" }
+    created_at { Time.current }
+    updated_at { Time.current }
```

Comment:
I don't think you need to supply these timestamps. A nice feature of ActiveRecord is how the `t.timestamps` method in a db migration gives you these columns, which update themselves on insert/update.